### PR TITLE
CC-28712 Introduce Shipment + Service Points IG

### PR DESCRIPTION
**Pull request by AsonUnique**
## PR Description

- Epic ticket: https://spryker.atlassian.net/browse/CC-28712
- IG ticket: https://spryker.atlassian.net/browse/CC-28724

### CODE REVIEW

**Review by abitskil — APPROVED** [review]

**Review by andriispryker — DISMISSED** [review]
Approved.

File: docs/scos/dev/feature-integration-guides/202304.0/install-the-shipment-service-point-feature.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Install the Shipment + Service Points feature
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202304.0/install-the-shipment-service-point-feature.md:2]: the included file's name, IG's name, and the title must be identic. In file names, it is "service-point" and the title says "service points"? Chose the proper one and adjust the title or file names accordingly.

Comment (AsonUnique) [reply docs/scos/dev/feature-integration-guides/202304.0/install-the-shipment-service-point-feature.md]: Fixed file names 👍

File: _includes/pbc/all/install-features/202304.0/install-the-shipment-service-points-feature.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+
+
+
+This document describes how to integrate the [Shipment](/docs/scos/user/features/{{site.version}}/shipment/shipment-feature-overview.html) + Service Points feature into a Spryker project.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-shipment-service-points-feature.md:4]:
```suggestion
This document describes how to integrate the [Shipment](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/shipment-feature-overview.html) + Service Points feature into a Spryker project.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**